lib/core/test/integration/Test/Integration/Framework/TestData.hs

```diff
@@ -220,6 +221,10 @@ errMsg403NotEnoughMoney has needs = "I can't process this payment because there'
     \ " ++ show has ++ " Lovelace, but I need " ++ show needs ++ " Lovelace\
     \ (excluding fee amount) in order to proceed  with the payment."
 
+errMsg403InvalidTransaction :: String
+errMsg403InvalidTransaction = "I can't process this payment because transactions\
+    \ with 0 amount are not supported in Byron."
```

Comment:
Wording suggestion:

> I can't process this payment because it contains at least one payment output of value 0. This isn't supported by the current core nodes.

Reply:
done